### PR TITLE
small edits to fix typos and improve readability

### PR DESCRIPTION
**Pull request by zootella**
Hi @RangerMauve, re our discussion a week or two ago, here's a pr of some stuff I found reading through the first section of the Dat documentation. Let me know what you think and if I should merge

### CODE REVIEW

File: docs/getting-started-faq.md [hunk 1]
```diff
@@ -93,11 +93,11 @@ dat sync /my-dat --http
 
 Once running, visit `localhost:8080` to view the latest content. Set the version flag, `localhost:8080/?version=3` to see a specific version. Clicking on a file will download that version of the file (assuming its available locally or on the network).
 
-We are working on adding a local version history backup in the command line and desktop application. The interfaces for using and checking out older versions will also be further developed.
+We are working on adding a local version history backup in the command line and desktop applications. We'll also further develop the interfaces for using and checking out older versions of files.
 
 ### Is there a JavaScript or Node.js implementation?
 
-Find it on GitHub: [dat-node](http://github.com/datproject/dat-node) for Node and [dat-js](http://github.com/datproject/dat-js) for other applications.
+Yes! Find them on GitHub: [dat-node](http://github.com/datproject/dat-node) for Node.js and [dat-js](http://github.com/datproject/dat-js) for other applications.
```
[the remark this answers is not in the document]
Comment (okdistribute) [reply docs/getting-started-faq.md]: yes please swap this for dat-sdk. x

File: docs/getting-started-installation.md [hunk 2]
```diff
@@ -30,20 +30,20 @@ Once the install finishes, you should be able to run the `$ dat` command in your
 
 ### Using npm
 
-If you have `npm`, you can install Dat with it:
```
[the remark this answers is not in the document]
Comment (okdistribute) [reply docs/getting-started-installation.md]: yes. also we should add links to the released binaries if that isn't already here

File: docs/getting-started-faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zootella) [reply docs/getting-started-faq.md]: Sounds good, changing to:

```
Yes! On GitHub, use the Dat [SDK](https://github.com/datproject/sdk).
```

File: docs/getting-started-installation.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zootella) [reply docs/getting-started-installation.md]: @okdistribute What are the links to the prebuilt binaries?

For now I'll just restore the original phrasing @RangerMauve

**Comment by RangerMauve** [comment]
@zootella Err, was the branch deletion on purpose? We didn't get a chance to merge it yet! :P

**Pull request closed without merging**